docs/program/extend/modular-resources/_index.md

```diff
@@ -1,150 +1,528 @@
 ---
 title: "Create custom components and services as modular resources"
 linkTitle: "Modular Resources"
+image: "/tutorials/img/intermode/rover_outside.png"
+imageAlt: "An intermode rover pictured outdoors."
 weight: 10
 type: "docs"
 tags: ["server", "rdk", "extending viam", "modular resources", "components", "services"]
+description: "Use the Viam module system to implement custom resources that can be included in any Viam-powered robot."
 ---
 
-The Viam module system allows you to integrate custom resources ([components](/components) and [services](/services)) into any robot running on Viam.
-Once configured, custom resources behave the same as built-in RDK resources.
-
-## Viam resource basics
+The Viam module system allows you to integrate custom [resources](/appendix/glossary/#term-resource) ([components](/components) and [services](/services)) into any robot running on Viam.
 
 With modular resources, you can:
 
-- Create new models of existing (RDK built-in) component or service types
+- Create new models of built-in component or service types
 - Create brand new resource types
 
-Modular resources that are configured and composed into a Viam robot behave identically to built-in resources.
-This means that functionality that the Viam [RDK](/internals/rdk/) provides for built-in resources is also automatically provided for user-created resources.
+`viam-server` [manages](#modular-resource-management) modular resources configured on your robot like resources that are already built-in to the [Robot Development Kit (RDK)](/internals/rdk).
+
+Two key concepts exist across all Viam resources, built-in and modular, to facilitate this: [*APIs*](#apis) and [*models*](#models).
 
-Two key concepts exist across all Viam resources (built-in and modular) to facilitate this: *APIs* and *models*
+## Key concepts
 
 ### APIs
 
-Every Viam component and service type exposes an *API*.
-This API describes the interface for the particular component or service type.
-For example, the API of built-in component type [camera](/components/camera) exposes methods such as *GetImage*, and the API of built-in service type [vision](/services/vision) exposes methods such as *GetDetectionsFromCamera*.
+Every Viam [resource](/appendix/glossary/#term-resource) exposes an [Application Programming Interface (API)](https://www.ibm.com/topics/api).
```

Comment:
[optional/opinion] IBM link is a weird place to go for a definition. Why not wikipedia or more obvious "reference" site?

Reply:
Ha true, will change to wiki